db/migrate/20130409133702_create_timelines_planning_element_types.rb

```diff
@@ -45,7 +45,7 @@ def self.up
     end
 
     add_index :timelines_planning_element_types, :color_id
-    add_index :timelines_planning_element_types, :project_type_id
+    add_index :timelines_planning_element_types, :project_type_id, :name => "idx_tlpet"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.